docs/onboarding/09-exposing-twins-over-graphql.md

```diff
@@ -0,0 +1,46 @@
+# Chapter 9: Exposing Digital Twins over GraphQL
+
+- [Why GraphQL?](https://www.apollographql.com/docs/intro/benefits/) - Explains the value proposition of GraphQL
+- [GraphQL Tutorial](https://www.apollographql.com/docs/tutorial/introduction/) - Walks through coding a GraphQL service. **STOP after [Chapter 4 (Write Mutation Resolvers)](https://www.apollographql.com/docs/tutorial/mutation-resolvers/)**. Apollo Explorer is not necessary to complete the tutorial. Browsing to localhost address provides means to execute requests.
+
+## Key Points
+

```

Comment:
Could we add a key point that summarizes value prop here?

Reply:
the value prop should be captured in the first two key points. also added a note that the video you suggested covers this.